src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    protected static final String HR = "    ____________________________________________________________\n";
```

Comment:
Should this be named as HORIZONTAL_ROW? HR seems a bit vague and might be mistaken for e.g. hour.